### ISSUE TITLE
Register Playback for Remote Commands

### ISSUE THREAD

**Issue by eximo84** [jellyfin/Swiftfin#565]
So when playing something the control other speakers / TVs doesn't show anything as playing on iOS.

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/25502878/189420193-87a8d398-e94c-420c-967b-8240ff3e7417.png)
</details>

Screenshot shows bbc iplayer previous played item not the current item playing via Swiftfin.

**Comment by LePips** [comment]
Yes, that's because we don't register to receive remote commands. I will convert this into a feature request for that.

**Comment by JPKribs** [comment]
Should be closed as part of https://github.com/jellyfin/Swiftfin/pull/1581
